tests/test_sklearn_scaler_converter.py

```diff
@@ -107,6 +109,33 @@ def test_float64_robust_scaler_floats(self):
         self.assertIsNotNone(torch_model)
         np.testing.assert_allclose(model.transform(data), torch_model.transform(data_tensor), rtol=1e-06, atol=1e-06)
 
+    # Tests with TVM backend
+    @unittest.skipIf(not tvm_installed(), reason="TVM test requires TVM installed")
+    def test_standard_scaler_floats_tvm(self):
+        # Generate a random 2D array with values in [0, 1000)
+        np.random.seed(0)
+        data = np.random.rand(100, 200) * 1000
+        data = np.array(data, dtype=np.float32)
+        data_tensor = torch.from_numpy(data)
+
+        model = StandardScaler(with_mean=False, with_std=False)
+        model.fit(data)
+        torch_model = hummingbird.ml.convert(model, "tvm", data, extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+        self.assertIsNotNone(torch_model)
+        np.testing.assert_allclose(model.transform(data), torch_model.transform(data_tensor), rtol=1e-06, atol=1e-06)
+
+        model = StandardScaler(with_mean=True, with_std=False)
```

Comment:
this test seems formatted differently than others.  due to size?

Reply:
mmm not sure, I see this formatted as the other tests.

Reply:
Ah, what I meant was usually you'd have `def _test_standard_scaler_floats` and  then you'd call  that function 3 times rather than having 3 calls to `StandardScaler`.  But it's small enough that it's fine!  just an observation.

Reply:
Ah got it. Yes, let me fix this.